calculator/src/extension.cpp

```diff
@@ -31,6 +32,7 @@ class Calculator::Private
 public:
     QPointer<ConfigWidget> widget;
     std::unique_ptr<mu::Parser> parser;
+    std::unique_ptr<mu::ParserInt> iparser;
```

Comment:
arithmetic hex operations are a extreme edge case. conversion fine, but virtually nobody will calculate hex values. I'd like to see this as an option.  besides, do you know the memory impact of such a parser?

Reply:
I don't know if I understood you, but I think changing it to be a NULL pointer and then actually creating it when someone requests "0x".

Reply:
I solved it using a check box in settings->extensions, changes in efcdf21e9fc7cdb086742e320c4802f259597137.